subsys/bluetooth/shell/bt.c

```diff
@@ -193,65 +193,84 @@ static bool data_cb(struct bt_data *data, void *user_data)
 	}
 }
 
-static void print_data_set(uint8_t type, uint8_t set_value_len,
-			   const uint8_t *scan_data, uint8_t scan_data_len)
+static void print_data_hex(const uint8_t *data, uint8_t len)
 {
-	uint8_t min_value_len = MIN(set_value_len, scan_data_len);
+	shell_fprintf(ctx_shell, SHELL_INFO, "0x");
+	for (int8_t i = len-1; i >= 0; i--) {
```

Comment:
```suggestion
	for (uint8_t i = len - 1; i >= 0; i--) {
```

Please also add a comment as why we are printing the largest index first and working our way down to `[0]`

Reply:
'i' is signed as we compare 'i >= 0' 

I'll do a comment on the reverse order